src/components/organisms/EntityDetails/EntityDetailsContent/Settings/SettingsTests/DelayModal.tsx

```diff
@@ -67,7 +74,7 @@ const DelayModal: React.FC<DelayModalProps> = props => {
           <Text className="regular middle">Delay in ms</Text>
           <InputNumber
             ref={delayInputRef}
-            placeholder="Delay"
+            placeholder="e.g.: 1000"
```

Comment:
Maybe worth to mention that we have human-readable syntax too, like `3m` or `1h`

Reply:
Yes, we have, but it is auto converted on BE. Definitely see this as an future improvement, but for now we don't have validation, etc. for this. May this be resolved in next stages?

Reply:
The validation matching both `1000` and `1000ms` or `1s` etc is as simple as `/^([0-9]|([1-9][0-9]+))(ms|s|m|h)?$/.test(value)` 🙂